### PR TITLE
Add JSON file for Current and Expected Logs for edx.course.tool.accessed event

### PR DESCRIPTION
**Pull request by imhassantariq**
**Trello Link:** [Here](https://trello.com/c/dDohuZmY/3-course-resource-event-edxcoursetoolaccessed)

**Description:** JSON file for Current and Expected Logs

**Checks before merge:**

- [x] JSON Reviewed
- [ ] Code Reviewed
- [x] Unit Test Pass
- [x] Commits squashed

### CODE REVIEW

**Comment by asamolion** [comment]
@imhassantariq Good work. The JSON looks fine to me. One small tip, Try to use a JSON formatter that sorts the keys alphabetically. That will make your files more consistent.

You can begin the work on the transformer now.

**Comment by asamolion** [comment]
@imhassantariq Please squash all the commits
@arooshaarif Please review the JSON.

File: openedx/features/caliper_tracking/transformers/navigation_transformers.py [hunk 1]
```diff
@@ -15,3 +15,44 @@ def edx_ui_lms_link_clicked(current_event, caliper_event):
...
+    caliper_object = {
+        'id': current_event['page'],
+        'type': 'WebPage',
+    }
+
+    caliper_event.update({
+        'type': 'ToolUseEvent',
+        'action': 'Used',
+        'object': caliper_object,
+
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/navigation_transformers.py]: Please remove unnecessary whitespace.

File: openedx/features/caliper_tracking/transformers/navigation_transformers.py [hunk 2]
```diff
@@ -15,3 +15,44 @@ def edx_ui_lms_link_clicked(current_event, caliper_event):
     :return: updated caliper_event.
     """
     return current_event
+
+
+def edx_course_tool_accessed(current_event, caliper_event):
+    """
+        The browser emits an edx.course.tool.accessed event when a user clicks
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/navigation_transformers.py]: Un-indent this to make it exactly below the above `"""`

File: openedx/features/caliper_tracking/transformers/navigation_transformers.py (hunk 1 above)
Comment (imhassantariq) [reply openedx/features/caliper_tracking/transformers/navigation_transformers.py]: fixed

File: openedx/features/caliper_tracking/transformers/navigation_transformers.py (hunk 2 above)
Comment (imhassantariq) [reply openedx/features/caliper_tracking/transformers/navigation_transformers.py]: fixed

**Comment by imhassantariq** [comment]
@asamolion rebased with develop. Please Review
